server.js

```diff
@@ -64,62 +66,64 @@ app.prepare().then(() => {
       // path /(help|chat) -> Discord chat invite
       res.writeHead(301, { Location: 'https://discordapp.com/invite/dvwXA2N' })
       res.end()
-    } else if (/^\/doc\/commands-reference(\/.*)?/.test(pathname)) {
-      // path /doc/commands-reference... -> /doc/command-reference...
+    } else if (/^\/(docs|documentation)(\/.*)?$/i.test(pathname)) {
+      // path /docs... or /documentation... -> /doc...
       res.writeHead(301, {
-        Location: req.url.replace('commands-reference', 'command-reference')
+        Location: req.url.replace(/\/(docs|documentation)/i, '/doc')
       })
       res.end()
-    } else if (/^\/doc\/tutorial\/(.*)?/.test(pathname)) {
-      // path /doc/tutorial/... -> /doc/tutorials/deep/...
+    } else if (/^\/doc\/commands-reference(\/.*)?$/i.test(pathname)) {
+      // path /doc/commands-reference... -> /doc/command-reference...
       res.writeHead(301, {
-        Location: req.url.replace('/doc/tutorial/', '/doc/tutorials/deep/')
+        Location: req.url.replace(
+          '/doc/commands-reference',
+          '/doc/command-reference'
+        )
       })
       res.end()
-    } else if (pathname === '/doc/tutorial' || pathname === '/doc/tutorial/') {
+    } else if (/^\/doc\/tutorial\/?$/i.test(pathname)) {
       // path /doc/tutorial -> /doc/tutorials
       res.writeHead(301, {
-        Location: req.url.replace('/doc/tutorial', '/doc/tutorials')
+        Location: req.url.replace(/\/doc\/tutorial\/?/, '/doc/tutorials')
+      })
+      res.end()
+    } else if (/^\/doc\/tutorial\/(.*)?/.test(pathname)) {
+      // path /doc/tutorial/... -> /doc/tutorials/deep/...
+      res.writeHead(301, {
+        Location: req.url.replace('/doc/tutorial/', '/doc/tutorials/deep/')
       })
       res.end()
     } else if (
-      pathname === '/doc/use-cases/data-and-model-files-versioning' ||
-      pathname === '/doc/use-cases/data-and-model-files-versioning/'
+      /^\/doc\/use-cases\/data-and-model-files-versioning\/?$/.test(pathname)
     ) {
       // path /doc/use-cases/data-and-model-files-versioning
       //  ->  /doc/use-cases/versioning-data-and-model-files
       res.writeHead(301, {
         Location: req.url.replace(
-          'data-and-model-files-versioning',
-          'versioning-data-and-model-files'
+          '/doc/use-cases/data-and-model-files-versioning',
```

Comment:
I don't think this prefix is needed. It should be doing exactly the same to my mind.

Reply:
k, I see this is consistency thingy